ktlint/src/test/kotlin/com/pinterest/ktlint/SimpleCLITest.kt

```diff
@@ -153,6 +172,21 @@ class SimpleCLITest {
             }
     }
 
+    @Test
+    fun `Issue 1793 - Given some code with an error and no patterns read in from stdin then return nothing `(
+        @TempDir
+        tempDir: Path,
+    ) {
+        CommandLineTestRunner(tempDir)
+            .run(
+                "too-many-empty-lines",
+                listOf("--patterns-from-stdin"),
+                stdin = ByteArrayInputStream(ByteArray(0)),
+            ) {
+                assertNormalExitCode()
```

Comment:
I would expect that 0 files are processed. This can be validated using the debug output.